File: source/api/recipients/create/_create.java
import java.util.HashMap;

import co.omise.model.Recipient;

Recipient recipient = Recipient.create(new HashMap<String, Object>() {
  {put("name", "Somchai Prasert");}
  {put("email", "somchai.prasert@example.com");}
  {put("type", "individual");}
  {put("bank_account", new HashMap<String, Object>() {
    {put("brand", "bbl");}
    {put("number", "1234567890");}
    {put("name", "SOMCHAI PRASERT");}
  });}
});

File: source/api/recipients/create/_create.py
import omise

omise.api_secret = 'skey_test_4xs8breq3htbkj03d2x'

recipient = omise.Recipient.create(
    name='Somchai Prasert',
    email='somchai.prasert@example.com',
    type='individual',
    bank_account=dict(
        brand='bbl',
        number='1234567890',
        name='SOMCHAI PRASERT'
    )
)